reactivesocket-core/src/main/java/io/reactivesocket/RequestHandler.java

```diff
@@ -81,7 +82,7 @@ public Builder withFireAndForget(final Function<Payload, Publisher<Void>> handle
             return this;
         }
 
-        public Builder withRequestChannel(final Function<Publisher<Payload> , Publisher<Payload>> handleRequestChannel) {
+        public Builder withRequestChannel(final BiFunction<Payload, Publisher<Payload> , Publisher<Payload>> handleRequestChannel) {
```

Comment:
If I'm reading this correctly, the server side publisher will also receive an initial payload like all the other interactions?